static/src/javascripts/projects/common/modules/identity/wizard.js

```diff
@@ -1,15 +1,82 @@
 // @flow
-/* eslint-disable no-underscore-dangle, react/sort-comp */
 
 import fastdom from 'lib/fastdom-promise';
+import { scrollTo } from 'lib/scroller';
 
 const completedClassname = 'manage-account-wizard--completed';
 const pagerClassname = 'manage-account-wizard__controls-pager';
-const stepClassname = 'manage-account-wizard__step';
 const nextButtonElClassname = 'js-manage-account-wizard__next';
 const prevButtonElClassname = 'js-manage-account-wizard__prev';
 const containerClassname = 'manage-account-wizard';
 
+const stepClassname = 'manage-account-wizard__step';
+const stepHiddenClassname = 'manage-account-wizard__step--hidden';
+const stepOutClassname = 'manage-account-wizard__step--out';
+const stepInClassname = 'manage-account-wizard__step--in';
+const stepOutReverseClassname = 'manage-account-wizard__step--out-reverse';
+const stepInReverseClassname = 'manage-account-wizard__step--in-reverse';
+const stepTransitionClassnames = [
+    stepInClassname,
+    stepInReverseClassname,
+    stepOutClassname,
+    stepOutReverseClassname,
+];
+
+const getDirection = (currentPosition: number, newPosition: number): string => {
+    if (currentPosition < 0) {
+        return 'none';
+    } else if (currentPosition > newPosition) {
+        return 'backwards';
+    }
+    return 'forwards';
+};
+
+const animateIncomingStep = (
+    wizardEl: HTMLElement,
+    stepEl: HTMLElement,
+    direction: string
+): Promise<void> =>
+    fastdom
+        .write(() => {
+            stepEl.classList.remove(
+                stepHiddenClassname,
+                ...stepTransitionClassnames
+            );
```

Comment:
Oh, sorry I haven't seen this earlier: there is a bug in the `classList` polyfill from pf.io, which prevents us from using multiple classNames in `add, remove`. These have to be in a loop of split up in seperate calls unfortunately.